src/components/NavDropdown.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
```

Comment:
@Chrismarcel have you heard of object destructuring, it could make the app a little lighter if we don't get everything in prop-types for each component that loads. This might come in handy `https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment`, `https://medium.freecodecamp.org/the-basics-of-destructuring-props-in-react-a196696f5477`